evap/student/views.py

```diff
@@ -162,9 +162,12 @@ def render_vote_page(request, evaluation, preview, for_rendering_in_modal=False)
         evaluation_form_group_bottom = []
 
     template_data = {
-        "errors_exist": any(
+        "contributor_errors_exist": any(
+            any(form.errors for form in form_group) for form_group in [*(form_groups.values())]
+        ),
```

Comment:
Why `[*(form_groups.values())]`? Could this just be
```suggestion
        "contributor_errors_exist": any(form.errors for form_group in form_groups.values() for form in form_group)
```
?